src/lightning_hivemind/strategy.py

```diff
@@ -22,14 +22,26 @@
 import torch
 from lightning_fabric.strategies.strategy import TBroadcast
 from lightning_fabric.utilities.types import LRScheduler, ReduceLROnPlateau
-from pytorch_lightning import Trainer
-from pytorch_lightning.strategies import Strategy
-from pytorch_lightning.utilities.data import extract_batch_size
-from pytorch_lightning.utilities.exceptions import MisconfigurationException
-from pytorch_lightning.utilities.model_helpers import is_overridden
-from pytorch_lightning.utilities.rank_zero import rank_zero_warn
+from lightning_utilities import module_available
 from torch import Tensor
 
+if module_available("lightning"):
```

Comment:
can we check for "lightning.pytorch" here?

Reply:
I think there is no way to have lighting without pytorch :)

Reply:
currently there isn't :)